### PR TITLE
Create Comments System

### PR DESCRIPTION
**Pull request by obieglok**
Created Comments System 

Once the comments form gets submitted it is sent to a server where it is stored. On the load of the page a fetch is sent to find any comments that have been on the server and displays them in the table on the comments page

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/CommentsClass.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Date;
+
+public final class CommentsClass{
```
Comment (marvel308) [inline portfolio/src/main/java/com/google/sps/data/CommentsClass.java:19]: Comments missing.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -45,9 +53,23 @@ function moreFacts(){
...
+function getComments(){
+/**
+ * Get's comments from the server and displays them in the table
+ * by creating new rows.
+ */
+  fetch('/data').then(response => response.json()).then((comments) => {
+  const displayTable = document.getElementById('table') ;
+  comments.forEach((comment) => {
+
+  let row = displayTable.insertRow(1);
```
Comment (marvel308) [inline portfolio/src/main/webapp/script.js:65]: indentation is wrong.

File: portfolio/src/main/java/com/google/sps/data/CommentsClass.java (hunk 1 above)
Comment (obieglok) [reply portfolio/src/main/java/com/google/sps/data/CommentsClass.java:19]: Added the comments now

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (obieglok) [reply portfolio/src/main/webapp/script.js]: Fixed it

**Pull request merged**